[2022-07-27 15:50:11,864] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dataquality_pipeline_v2.create_snowflake_stage manual__2022-07-27T15:49:45.667788+00:00 [queued]>
[2022-07-27 15:50:11,886] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dataquality_pipeline_v2.create_snowflake_stage manual__2022-07-27T15:49:45.667788+00:00 [queued]>
[2022-07-27 15:50:11,886] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-27 15:50:11,886] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-07-27 15:50:11,886] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-27 15:50:11,920] {taskinstance.py:1397} INFO - Executing <Task(SnowflakeOperator): create_snowflake_stage> on 2022-07-27 15:49:45.667788+00:00
[2022-07-27 15:50:11,939] {standard_task_runner.py:52} INFO - Started process 279 to run task
[2022-07-27 15:50:11,938] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dataquality_pipeline_v2', 'create_snowflake_stage', 'manual__2022-07-27T15:49:45.667788+00:00', '--job-id', '134', '--raw', '--subdir', 'DAGS_FOLDER/dataquality_pipeline_v2.py', '--cfg-path', '/tmp/tmpg41jo779', '--error-file', '/tmp/tmp378242gv']
[2022-07-27 15:50:11,948] {standard_task_runner.py:80} INFO - Job 134: Subtask create_snowflake_stage
[2022-07-27 15:50:12,100] {task_command.py:371} INFO - Running <TaskInstance: dataquality_pipeline_v2.create_snowflake_stage manual__2022-07-27T15:49:45.667788+00:00 [running]> on host 2c2de32fb47f
[2022-07-27 15:50:12,312] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=abin404@yopmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dataquality_pipeline_v2
AIRFLOW_CTX_TASK_ID=create_snowflake_stage
AIRFLOW_CTX_EXECUTION_DATE=2022-07-27T15:49:45.667788+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-07-27T15:49:45.667788+00:00
[2022-07-27 15:50:12,313] {snowflake.py:118} INFO - Executing: CREATE OR REPLACE STAGE YELLOW_TRIPDATA_STAGE url=s3://snowflakedata212
credentials=(aws_key_id='AKIA3LREOZJOXQMV7ZUN' aws_secret_key='***')
file_format=(type = 'CSV', skip_header = 1, time_format = 'YYYY-MM-DD HH24:MI:SS');
[2022-07-27 15:50:12,329] {base.py:68} INFO - Using connection ID 'snowflake_default' for task execution.
[2022-07-27 15:50:12,337] {connection.py:262} INFO - Snowflake Connector for Python Version: 2.7.9, Python Version: 3.7.13, Platform: Linux-5.15.0-1014-azure-x86_64-with-debian-11.4
[2022-07-27 15:50:12,338] {connection.py:877} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2022-07-27 15:50:12,338] {connection.py:894} INFO - Setting use_openssl_only mode to False
[2022-07-27 15:50:14,011] {cursor.py:710} INFO - query: [ALTER SESSION SET autocommit=True]
[2022-07-27 15:50:14,293] {cursor.py:734} INFO - query execution done
[2022-07-27 15:50:14,293] {snowflake.py:328} INFO - Running statement: CREATE OR REPLACE STAGE YELLOW_TRIPDATA_STAGE url=s3://snowflakedata212
credentials=(aws_key_id='AKIA3LREOZJOXQMV7ZUN' aws_secret_key='***')
file_format=(type = 'CSV', skip_header = 1, time_format = 'YYYY-MM-DD HH24:MI:SS');, parameters: None
[2022-07-27 15:50:14,294] {cursor.py:710} INFO - query: [CREATE OR REPLACE STAGE YELLOW_TRIPDATA_STAGE url=s3://snowflakedata212 credenti...]
[2022-07-27 15:50:15,567] {cursor.py:734} INFO - query execution done
[2022-07-27 15:50:15,568] {snowflake.py:338} INFO - Statement execution info - {'status': 'Stage area YELLOW_TRIPDATA_STAGE successfully created.'}
[2022-07-27 15:50:15,568] {snowflake.py:342} INFO - Rows affected: 1
[2022-07-27 15:50:15,568] {snowflake.py:343} INFO - Snowflake query id: 01a5e3b6-0000-da7c-0000-2259000625fa
[2022-07-27 15:50:15,568] {connection.py:507} INFO - closed
[2022-07-27 15:50:15,798] {connection.py:510} INFO - No async queries seem to be running, deleting session
[2022-07-27 15:50:16,173] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=dataquality_pipeline_v2, task_id=create_snowflake_stage, execution_date=20220727T154945, start_date=20220727T155011, end_date=20220727T155016
[2022-07-27 15:50:16,295] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-27 15:50:16,535] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
